src/cv/yolo_video_detect.py

```diff
@@ -0,0 +1,114 @@
+import cv2
+import imutils
+import numpy as np
+import os
+import time
+
+def get_configs(args):
+    """ Retrieves the object detector model resources """
+    labelsPath = os.environ.get('YOLO_NAMES_PATH')
+    labels = open(labelsPath).read().strip().split("\n")
+
+    np.random.seed(int(time.time()))
+    colors = np.random.randint(0, 255, size=(len(labels), 3), dtype="uint8")
+
+    weightsPath = os.environ.get('YOLO_WEIGHTS_PATH')
+    configPath = os.environ.get('YOLO_CONFIG_PATH')
```

Comment:
What happens if the environment variables doesn't exist in the OS or change all of a sudden in the OS? I suggest that you make a settings.py and make them as constants in that file and set defaults or errors if those variables don't exist.

Reply:
Added a settings file and added exceptions